yarn.lock

```diff
@@ -5687,6 +5687,14 @@ react-router@^4.2.0:
     prop-types "^15.5.4"
     warning "^3.0.0"
 
+react-sane-contenteditable@ethanlee16/react-sane-contenteditable:
```

Comment:
Looks like there's a merge conflict here, do you want to resolve it on your end?

Reply:
not a merge conflict! this is a package resolved by a [Git repository](https://github.com/ethanlee16/react-sane-contenteditable)

Reply:
Oh weird GitHub is saying “this branch has conflicts that must be resolved” in yarn.lock, will this get dealt with automatically?

Reply:
I see that, and yes- I will rebase this before merging